### DISCUSSION TITLE
Doesn't work

### DISCUSSION THREAD

**Discussion by ghost** [Discord-Toolkit-Official/Discord-Toolkit-Public#6] [Q&A] [answered]
I get modal that says it is 1.0 Release with "Test" button, but UI doesn't show

**Comment by TRH-Raz** [comment]
Project is dead.

**Comment by Raz-js** [comment, marked as answer]
Project is officially dead, we are working on a new version, thanks!

- Edit: for more information on why its not working, its because discord updated their api!

**Reply by ghost** [reply]
Oh.. on that case, is there any information about the new version?

**Reply by Raz-js** [reply]
@FenixGlobElite, yeah, owner of this org's account was hacked, we've updated and improved a new project at https://github.com/DSCHX/DCHX

**Reply by ghost** [reply]
Thanks for replying, I really waited for this

**Reply by Raz-js** [reply]
> Thanks for replying, I really waited for this

Apologies for the late response, just would like to ask what OS you use, as of now I am only allowed to see the windows file system of discord so if you happen to use linux or macos it would be really helpful

**Reply by ghost** [reply]
No problem. At the moment I use Windows. If I switch OS, I don't mind to try this and give you feedback.

**Reply by Raz-js** [reply]
> No problem. At the moment I use Windows. If I switch OS, I don't mind to try this and give you feedback.

Oh aight, join up if u want
https://discord.gg/44UcvQuG
